bin/get_tokens

```diff
@@ -5,35 +5,30 @@ require 'base64'
 require 'dotenv'
 require 'httparty'
 
-dotenv_file_name = '.env'
+DOTENV_FILE_NAME = '.env'
+OAUTH_ENDPOINT = 'https://apps.fortnox.se/oauth-v1'
 
-Dotenv.load(dotenv_file_name)
+Dotenv.load(DOTENV_FILE_NAME)
 
-print "Let's get you an access token and a refresh token for Fortnox API."
-
-client_id = ENV.fetch('FORTNOX_API_CLIENT_ID')
-client_secret = ENV.fetch('FORTNOX_API_CLIENT_SECRET')
-redirect_uri = ENV.fetch('FORTNOX_API_REDIRECT_URI')
-scopes = ENV.fetch('FORTNOX_API_SCOPES')
-credentials = Base64.encode64("#{client_id}:#{client_secret}").to_s
-oauth_endpoint = 'https://apps.fortnox.se/oauth-v1'
+CLIENT_ID = ENV.fetch('FORTNOX_API_CLIENT_ID'){ |key| raise "Missing environment variabel #{key}! Exiting." }
```

Comment:
Is this allowed? I would think that this would cause a "Dynamic assignment to constant" error ...

Reply:
Yes, it works just as intended.